lib/KettleMiddleware-Multer.js

```diff
@@ -0,0 +1,153 @@
+/*!
+Kettle wrapping for Multer Express Middleware
+
+Copyright 2017-2018 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/kettle/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+// Wraps the standard Express Multer middleware, for handling
+// multipart/form-data (primarily for file uploads)
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle");
+
+fluid.require("multer", require, "kettle.npm.multer");
+
+fluid.defaults("kettle.middleware.multer", {
+    gradeNames: "kettle.plainAsyncMiddleware",
+    // See https://github.com/expressjs/multer#multeropts;
+    // because Multer expects to receive functions for some
```

Comment:
Although it expects to receive functions we should do our best to unwrap these back to plain data as a courtesy for implementors